packages/shared/src/utils/constants/contract.ts

```diff
@@ -10,7 +10,9 @@ import { LocalStore } from '../storage';
 export const getLocalizedBasis = () =>
     ({
         accumulator: localize('Accumulators'),
+        current_stake: localize('Current stake'),
```

Comment:
I noticed we reached a point at which we keep declaring new objects with the same variables  for the same strings all over again across the app and later we have to extend it like here. Some of the strings in this function have been redeclared in 3-4 different objects and refer to identical text. While it was a good idea in the beginning, I think since having these variables ends up serving no purpose, we need to remove it to avoid further redeclaration across dtrader

Reply:
But this is existing object from production, which served the same purpose - to store basis. I didn't declare a new object.
What exactly do you suggest?

Reply:
This object is used for displaying information on the buttons (on production and on DTrader-V2)